andbible/beta/mods.d/NinetyDaysReadingPlan.conf

```diff
@@ -0,0 +1,10 @@
+[NinetyDaysReadingPlan]
+DataPath=./modules/texts/ztext/NinetyDaysReadingPlan/
+Version=1.0
+Description=Through the Bible in 90 days
+ShortPromo=Challenge yourself and read through the entire Bible in 90 days
+DistributionLicense=Public Domain
+Category=And Bible
+ModDrv=RawGenBook
+Versification=KJV
+AndBibleProvidesReadingPlan=NinetyDaysReadingPlan.properties
```

Comment:
Please add AndBibleMinimumVersion (it will be read by AndBibleAddonFilter, so the module is not displayed to versions that do not support it yet)

Reply:
Done

Reply:
If I saw correctly, you did not change the zip file (that contains a copy of this conf)? I think it should be applied there too. Though ... I agree it does not affect anything anymore. Okay I'll merge now.

Reply:
Sorry, did it when I should be sleeping 😴 can fix properly tomorrow